queries.yaml

```diff
@@ -231,12 +237,11 @@ post_process:
       property_keys: [name, kind]
       geometry_types: [Point]
       min_distance: 256.0
-  - fn: TileStache.Goodies.VecTiles.transform.drop_features_where
+  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
```

Comment:
This filter was supposed to drop any building polygons from the `landuse` layer, but will now just drop the `mz_is_building` property and keep the feature. Perhaps this should be changed to?

```yaml
  - fn: TileStache.Goodies.VecTiles.transform.drop_features_where
    params:
      source_layer: landuse
      where: >-
        properties.get('mz_is_building') == True
```